sparse/file.go

```diff
@@ -220,10 +220,5 @@ func GetFiemapRegionExts(file FileIoProcessor, interval Interval) ([]Extent, err
 		lastExtStart = ext.Logical
 	}
 
-	// last ext should have the FIEMAP_EXTENT_LAST set, otherwise we don't get all exts
-	if exts[len(exts)-1].Flags&FIEMAP_EXTENT_LAST == 0 {
```

Comment:
Why FIEMAP_EXTENT_LAST can be unset? In other way, why we have to ensure it's set before?